src/main/react-front-end/src/views/Login/UserAccountComponent.js

```diff
@@ -138,6 +139,22 @@ export default class UserAccountComponent extends Component {
 			if (currentSaveStatus) {
 				// if the user opted to switch from saving tokens on browser to
 				// storing tokens on the server, we clear all saved tokens in the current browser session.
+				let credentials = []
+				if(!(typeof cookies.get(GOOGLEDRIVE_NAME) == "undefined")){
+					var googleDriveCredentials = JSON.parse(cookies.get(GOOGLEDRIVE_NAME));
+					googleDriveCredentials.forEach(function(element){
+						element.name = "GoogleDrive: " + element.name;
+					});
+					credentials.push(...googleDriveCredentials);
+				}
+				if(!(typeof cookies.get(DROPBOX_NAME) == "undefined")){
+					var dropBoxCredentials = JSON.parse(cookies.get(DROPBOX_NAME));
+					dropBoxCredentials.forEach(function(element){
+						element.name = "Dropbox: " + element.name;
+					});
+					credentials.push(...dropBoxCredentials);
+				}
+				saveOAuthCredentials(credentials, (success)=>{console.log("Credentials saved Successfully")}, (error)=>{console.log("Error in saving credentials")});
```

Comment:
On error it will be better if we trigger an eventEmitter and inform the user that there was an error in storing the existing credentials and they may have to reauthenticate their accounts in again in future.